recipes/libtommath/cmake/patches/0002-enable-building-dlls.patch

```diff
@@ -0,0 +1,87 @@
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -91,6 +91,14 @@ add_library(${PROJECT_NAME}
+     ${HEADERS}
+ )
+ 
++if(BUILD_SHARED_LIBS AND MSVC)
++  target_sources(${PROJECT_NAME} PRIVATE "tommath.def")
++endif()
++
++if(WIN32 AND NOT MSVC)
++  target_link_libraries(${PROJECT_NAME} PRIVATE crypt32)
++endif()
++
+ target_include_directories(${PROJECT_NAME} PUBLIC
+     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+     $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}>
+--- s_mp_rand_platform.c
++++ s_mp_rand_platform.c
```

Comment:
This part of the patch is not related to generating DLLs. I would not include if possible, instead, I recommend you sending it as a patch to the upstream.